models/models.go

```diff
@@ -65,6 +65,7 @@ type PipelineModel struct {
 	StatusReportName string                             `json:"status_report_name,omitempty" yaml:"status_report_name,omitempty"`
 	Stages           []StageListItemModel               `json:"stages,omitempty" yaml:"stages,omitempty"`
 	Workflows        GraphPipelineWorkflowListItemModel `json:"workflows,omitempty" yaml:"workflows,omitempty"`
+	Priority         *int                               `json:"priority,omitempty" yaml:"priority,omitempty"`
```

Comment:
Priority has an integer pointer type, to distinguish between priority 0 and not set priority.

Reply:
Would not it be better if we set a default priority to every workflow? What if we say that the default is 0 because that is right in the middle of the -100 - 100 scale. This way we do not need to work with pointers and worry if a workflow has a priority or not.

(This comment also applies to the structs in the selective_trigger.go file.)

Reply:
The thing is that priorities can override each other at different levels. For example, a trigger item's priority can override a workflow's default priority.
Since the backend side handles priority, this implementation lets the backend define override logic the way it needs.